data/org.freedesktop.portal.Notification.xml

```diff
@@ -53,37 +53,111 @@
 
         The ID can be used to later withdraw the notification.
         If the application reuses the same ID without withdrawing,
-        the notification is replaced by the new one.
+        the notification is updated with the new one. It's possible
+        to set a ``display-hint`` to animate replacing the notification
+        instead of updating it.
 
         The format of the serialized notification is a vardict, with
         the following supported keys, all of which are optional:
 
+        * ``application-id`` (``s``)
+
+          An optional application ID for Flatpak applications, otherwise it's mandatory.
+          This has to be the Flatpak application ID or a desktop file ID, with optionally a postfix.
+          There is no reason to set this property if the application uses Flatpak and no postfix is set.
+          E.g ``org.libreoffice.LibreOffice.Writer`` where
+          ``org.libreoffice.LibreOffice`` is the flatpak application ID.
+          If this property isn't set it will be picked up automatically, if possible.
+
+          See `Application ID <https://flatpak-docs.readthedocs.io/en/latest/conventions.html>`_ and
+          `Desktop File ID <https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s02.html#desktop-file-id>`_
+
         * ``title`` (``s``)
 
           User-visible string to display as the title.
+          The title should be a short string, if is too
+          long it may not be fully displayed.
 
         * ``body`` (``s``)
 
-          User-visible string to display as the body.
+          User-visible string to display as the body of the notification.
+          This can be a long description but note that if it is too long
+          it may not be fully displayed.
+
+          Any markup is removed from the string including new lines.
```

Comment:
This makes it sound like if the client gives a string like `<i>foobar</i>` the notification server will just render `foobar` (just without making it italics). That's not what we want, though. We want an app to say `<i>foobar</i>` and the server should render `<i>foobar</i>`

Reply:
the idea is to filter out markup like [FDO notifications](https://specifications.freedesktop.org/notification-spec/notification-spec-latest.html#markup)  requires servers to do.

> That's not what we want, though.

@AdrianVovk Any reason why you say that we don't want that?

Reply:
i actually implemented it already, but this sounds like it makes a lot of sense to pass `body` unseen.